src/bilder/images/dagster/templates/docker-compose.yaml.j2

```diff
@@ -72,3 +73,23 @@ services:
       - "traefik.http.middlewares.traefik-forward-auth.forwardauth.address=http://traefik-forward-auth:4181"
       - "traefik.http.middlewares.traefik-forward-auth.forwardauth.authResponseHeaders=X-Forwarded-User"
       - "traefik.http.services.traefik-forward-auth.loadbalancer.server.port=4181"
+  # per node healthcheck container
+  health-check:
+    image: 610119931565.dkr.ecr.us-east-1.amazonaws.com/dockerhub/mitodl/ol-infra-health-checks:latest
+    container_name: health-check
+    ports:
+    - "8097:8907"
+    # volumes:
+    # do I need this? Didn't need to specify when testing locally with ol-data-platform
+    - /var/run/docker.sock:/var/run/docker.sock
+    - /etc/docker/compose:/etc/docker/compose
+    labels:
+      # Explicitly tell Traefik to expose this container
+      - "traefik.enable=true"
+      # Send all health_check host traffic to a router
+      - "traefik.http.routers.healthcheck.rule=Host(`0.0.0.0`)"
```

Comment:
Probably looking for something like 

```
Host(`${DAGSTER_HOSTNAME}`) && PathPrefix(`/healthcheck`))
```

And I think you can omit the two priority lines entirely because those are for situations when it is possible that multiple routes will match an incoming request and you need to explicitly state that one is more important than the other. In this case, The addition of `PathPrefix` will always put the healthcheck step ahead of a the earlier step. If the request doesn't have that prefix, it goes to the earlier step that is just `Host`.